Header header
sensor_msgs/Image color_image
sensor_msgs/Image depth_image
int64 number
int64 type